superset/assets/src/components/TableSelector.jsx

```diff
@@ -130,7 +130,12 @@ export default class TableSelector extends React.PureComponent {
           `${encodeURIComponent(schema)}/${encodeURIComponent(substr)}/${forceRefresh}/`);
        return SupersetClient.get({ endpoint })
         .then(({ json }) => {
-          const filterOptions = createFilterOptions({ options: json.options });
+          const filterOptions = createFilterOptions({
+              options: json.options.map(o => ({
```

Comment:
@khtruong, I think the indentation here is wrong and will probably trip the linter.

Reply:
Hmm. I ran the linter and it was green. Do you prefer it formatted differently?

Reply:
I don't think the linter actually checks for this (maybe we should add it) but in general the JS code uses 2 space indents

Reply:
Ohhh. Brain fart. I wasn't seeing what you meant. Yes, let me update this to 2 spaces.